workshopapps/engineerprofile360.web
# Category Controller

#### Task reference
This section contains a link to a corresponding task in **Linear**.
#### Summary
This section contains a brief description of **_what was accomplished_** within the current PR.

Made sure all methods in category controller is working as supposed
#### Implementation details
This section describes the **_technical aspects_** of the solution, implemented within the current PR, that are worth mentioning to the reviewers.
#### How to test
This section contains a checklist, describing basic scenarios that would help verify that the submitted code works correctly.
[how to test](https://docs.google.com/document/d/1o7uAzKyb9wT_lvcG94l_Zo8_imPcFFEcn66KfraWbkA/edit?usp=sharing)
#### References 
Any additional information (screenshots, video, diagrams) that would help understand and review the PR.
![Screen Shot 2022-12-06 at 11 58 15 AM](https://user-images.githubusercontent.com/88077456/205920206-d4d3cc1b-d5ef-4b7f-856e-b8dc19f54e7c.png)

![Screen Shot 2022-12-06 at 11 56 21 AM](https://user-images.githubusercontent.com/88077456/205920281-0f542c34-fc45-483a-b977-7d933b803958.png)

![Screen Shot 2022-12-06 at 6 42 48 AM](https://user-images.githubusercontent.com/88077456/205921259-51b1aac4-f5b7-4902-80d6-f0bd8a386439.png)

![Screen Shot 2022-12-06 at 2 12 05 PM](https://user-images.githubusercontent.com/88077456/205921800-96bab976-2c25-4f8c-be28-c9e7d13f6951.png)

![Screen Shot 2022-12-06 at 2 06 52 PM](https://user-images.githubusercontent.com/88077456/205921122-f993d01b-0d05-4777-ab80-29e019de0765.png)
